tensorflow/core/kernels/mkl/BUILD

```diff
@@ -318,6 +318,7 @@ tf_cc_test_mkl(
     srcs = ["mkl_fused_batch_norm_op_test.cc"],
     linkstatic = 1,
     deps = [
+        ":mkl_conv_op",
```

Comment:
FYI - this change is added in the last commit but not in the original commits, since the original commits were done a while back, and the baseline changes of moving mkl kernels causes the need to adding this dependency.

Reply:
Thank you for the note!